spec/jobs/problem_check_spec.rb

```diff
@@ -11,6 +11,7 @@
     before do
       ::ProblemCheck::TestCheck =
         Class.new(::ProblemCheck) do
+          self.perform_every = 30.minutes
```

Comment:
Need these here because no checks without a `perform_every` will end up in this job.